app/serializers/articles_index_serializer.rb

```diff
@@ -0,0 +1,7 @@
+class ArticlesIndexSerializer < ActiveModel::Serializer
+  attributes :id, :title, :lead, :body, :created
```

Comment:
According to your PT story you are not returning body and that is also the assumption that your frontend team has. The serializer is for not sending unused data.